src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/BasePreRenderer.js

```diff
@@ -38,14 +38,16 @@ const BasePreRenderer = forwardRef((props, ref) => {
                     action,
                     checkIfContextMenuActive,
                 }) => (
-                    <Pressable
+                    <TouchableWithoutFeedback
```

Comment:
We should comment on this change, and why we change from Pressable to TouchableWithoutFeedback.

Reply:
I don't see why this is necessary. Readers won't have context about Pressable in the first place.

Reply:
Marking this as resolved for now